dist/tools/ethos/start_network_osx.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+ETHOS_DIR="$(cd "$(dirname "$0")" && pwd -P)"
```

Comment:
I think we should integrate this into one script, ie. `start_network.sh` and hide the diffs between macOS and Linux from the user in there.

Reply:
I tried that approach but I actually wanted to have a different script for the sake of clearness. IMHO it is better to be aware when we launch a linux or a OS X script, moreover, in bash is a bit more hard to read all the needed if's for the unification.

Reply:
Actually the reason is stated in the description of this PR.